### PR TITLE
Story1 display collection

### PR DESCRIPTION
**Pull request by ros-a**
Hi! I think I've finished story 1 :)

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+function printInsect (array $dbRow) : string {
```
Comment (norayna) [inline functions.php]: remove the space before the :
`function printInsect (array $dbRow): string {`

File: test/Functions.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        $expectedOutput = '<div class="insects"><img src="images/jewel_bug.jpeg"><h2>Jewel Bug</h2>' .
+            '<p class="species">Scutiphora Pedicellata</p><p>Date spotted: 2018-03-15</p>' .
+             '<p>Location spotted: India</p><p>Size: 5-20 mm</p></div>';
+        $actualOutput = printInsect($input);
+        $this->assertEquals($expectedOutput, $actualOutput);
+    }
+    public function testFailurePrintInsect()
+    {
+        $input = ['id' => '1', 'name' => 'Jewel Bug', 'species' => 'Scutiphora Pedicellata', 'date_spotted' => '2018-03-15', 'location_spotted' => 'India', 'size' => '5-20 mm', 'image_path' => 'images/jewel_bug.jpeg'];
+        $expectedOutput = 'I\'m sorry, that array doesn\'t contain the expected keys';
```
Comment (norayna) [inline test/Functions.php]: this sentence should have . in order to match exactly with the error message that the function returns

File: test/Functions.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        $input = [
+            'id' => 1,
+            'common_name' => 'Jewel Bug',
+            'species' => 'Scutiphora Pedicellata',
+            'date_spotted' => '2018-03-15',
+            'location_spotted' => 'India',
+            'size' => '5-20 mm',
+            'image_path' => 'images/jewel_bug.jpeg'];
+        $expectedOutput = '<div class="insects"><img src="images/jewel_bug.jpeg"><h2>Jewel Bug</h2>' .
```
Comment (norayna) [inline test/Functions.php]: need to have alt attribute in the expected output

File: test/Functions.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?php
+
```
Comment (norayna) [inline test/Functions.php]: need to rename this file to "functions.php" - it should have the exact same name as the file to be tested, including starting with lowercase

File: index.php [hunk 5]
```diff
@@ -1,64 +1,49 @@
...
+        </section>
+        <section class="container-insects">
+         <?php
+              foreach ($allInsects as $insect) {
+                   echo printInsect($insect);
+              }
+         ?>
+        </section>
+        <form id="add-insect-form" class="add-insect-form">
+        </form>
```
Comment (norayna) [inline index.php]: this shouldn't really be here until story 2 - imagine if you don't get to finish story 2 and you deploy story 1 with bits of code that don't do anything - not great to have that code in

File: index.php [hunk 6]
```diff
@@ -1,64 +1,49 @@
...
+        <h1>
+           <span class="title-insect">INSECT</span>
+           <span class="title-collector">COLLECTOR</span>
+        </h1>
+        <section class="add-insect-option">
+            <p>Did you spot a new insect?!</p>
+            <a href="#add-insect-form">
+                <button class="add-now-button">add now!</button>
+            </a>
+        </section>
```
Comment (norayna) [inline index.php]: this shouldn't really be here until story 2 - imagine if you don't get to finish story 2 and you deploy story 1 with bits of code that don't do anything - not great to have that code in

**Review by norayna — CHANGES_REQUESTED** [review]
looking good!

**Comment by ros-a** [comment]
Thanks for the comments! I've made those changes, you should be able to see them :)

**Review by norayna — APPROVED** [review]
🪲

**Pull request merged**